### PR TITLE
chore(benchmarks): add development cluster setup

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

Adds instructions on how to setup a development cluster from scratch. I ran into this issue when migrating from Helm 2 to 3, where I had no idea how the cluster had been set up and we didn't keep track of everything in the repository.

This also now accurately reflects how the cluster is setup, and additionally improves things a little bit:

- Grafana now has a persistent disk
- Grafana now automatically provisions the Zeebe dashboard
- Option to install Istio
- Option to install linkerd

## Pull Request Checklist

- [x] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [x] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [x] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

File: benchmarks/cluster/prometheus-operator-values.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+alertmanager:
+  enabled: false
+
+grafana:
+  adminPassword: camunda
```
Comment (Zelldon) [inline benchmarks/cluster/prometheus-operator-values.yml:5]: Thats why I havent added it the last time to the benchmark folder :sweat_smile:

**Review by ChrisKujawa — CHANGES_REQUESTED** [review]
Hey @npepinpe as far as I can tell it looks good haven't tried it. I assume you did?

Could we please move this into the doc/setup folder, this was the intended place for this.

https://github.com/zeebe-io/zeebe/blob/develop/benchmarks/docs/setup/README.md

And maybe remove then on the old setup readme the tiller install stuff etc.

File: benchmarks/cluster/prometheus-operator-values.yml (hunk 1 above)
Comment (npepinpe) [reply benchmarks/cluster/prometheus-operator-values.yml:5]: Yeah I thought about it, but it's not like it's a good password anyway. You'd still need to know the private IP of our Grafana instance. We could also always make it private again and you'd need to port-forward it, meaning you'd need access to the cluster, so really even having a password is kind of pointless :sweat_smile:

**Comment by npepinpe** [comment]
Hm, I figured this was only partially docs, since some of it is the infrastructure itself (e.g. the manifest), but definitely the README is docs. :thinking:

**Comment by npepinpe** [comment]
I moved the cluster/ stuff to the docs/setup/ folder, and removed docs/kubernetes/ as I'm not sure it's really useful anymore? It seemed to be redundant information that was already present in other documents (mostly the main README), and we don't use kustomize anymore anyway, so it was quite outdated.

**Review by ChrisKujawa — APPROVED** [review]
thanks :+1:

File: benchmarks/docs/setup/README.md [hunk 2]
```diff
@@ -1,29 +1,90 @@
...
+
+## Monitoring
+
+To monitor your cluster, we will set up a the [Prometheus Operator helm chart](https://github.com/helm/charts/tree/master/stable/prometheus-operator),
+which will install the following components the operator and Grafana.
+
+To do so, run:
+
+```sh
+helm install metrics stable/prometheus-operator --atomic
```
Comment (npepinpe) [inline benchmarks/docs/setup/README.md:35]:
```suggestion
helm install metrics stable/prometheus-operator --atomic -f prometheus-operator-values.yml
```

**Pull request merged**